packages/gatsby-cli/src/init-starter.js

```diff
@@ -108,6 +109,13 @@ type InitOptions = {
 module.exports = async (starter: string, options: InitOptions = {}) => {
   const rootPath = options.rootPath || process.cwd()
 
+  if (urlRegex({ exact: true }).test(rootPath)) {
```

Comment:
maybe the built in `url` module would work as well? the parsing would be more robust, maybe not tho

Reply:
I like using small, focused modules to serve my purpose. In this case, I saw it fit to use `url-regex`. I've used it before, and it does a great job of handling the complexity that comes with URL parsing _and_ URL validation. I say that because I'm not very good with regex 😅

Reply:
require(`url`).parse(string) might be better for our purposes. I'm ok with new small dependencies. Far better than maintaining our own utility module. But since it's built-in 🤷‍♂️ It'd be good to a) parse the name then b) check if the protocol and host are set.